templates/help/x-axis-grouping.html.twig

```diff
@@ -10,10 +10,10 @@ keys and labels gets swapped and in a graph the labels will form the legend.</p>
 
 <table>
     <tr>
-        <th>Size</th>
-        <th>Red</th>
-        <th>Green</th>
-        <th>Blue</th>
+        <th width="25%">Size</th>
```

Comment:
assuming this could not be done within the css you added?

Reply:
I think width is deprecated HTML attribute